src/random.cpp

```diff
@@ -164,20 +164,13 @@ void GetOSRand(unsigned char *ent32)
      * error if more are requested.
      * The call cannot return less than the requested number of bytes.
      */
-    #if defined(MAC_OSX)
-    // Fallback for OSX < 10.12
```

Comment:
Keeping this comment would make sense (so those reading the code understand why the check is there)